### PR TITLE
add ja pattern

### PR DESCRIPTION
**Pull request by floatingsun**
add new ja pattern to find more image #4

### CODE REVIEW

File: py_workers/wiki2itl.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 import argparse
 
-ja_patterns = [r'\[\[:(?:ファイル|画像):(.*?)\|',r'\[\[(?:ファイル|画像):(.*?)\|'] 
+ja_patterns = [r'\[\[:(?:ファイル|画像):(.*?)\|',r'\[\[(?:ファイル|画像):(.*?)\|', r'\[\[:(?:ファイル|画像):(.*?)\]\]']
```
Comment (t0-0) [inline py_workers/wiki2itl.py]: I believe that with the additions you have made, "[[...]]" is now recognisable! Thank you.

However, only one of these patterns is supposed to be used, which is problematic. So I think they should be combined into one.
For example, "[[...]]" and "[[...|...]]" are both on the same page.

~Also, "]]" should be recognised by a regular expression. This saves the hassle of having to exclude trailing "thumb" etc. later.~

Comment (floatingsun) [reply py_workers/wiki2itl.py]: Make sense, could you pls make some change on the "add_ja_pattern" branch for that ?

Comment (t0-0) [reply py_workers/wiki2itl.py]: Changed, please confirm. If it is not good, please change it back.

**Review by floatingsun — COMMENTED** [review]
LGTM

**Pull request merged**